src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+// Actions
+const CHECK_STATUS = 'bookstore/categoriesReducer/CHECK_STATUS';
+
+// Reducer
+export default function categoriesReducer(state = [], action = {}) {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return 'Under Construction';
+    default: return state;
+  }
+}
+
```

Comment:
- [ ] Kindly make sure that `under construction` is displayed when check status button is clicked.

https://user-images.githubusercontent.com/70269507/174943981-6894ced6-cdd9-4c74-ad27-e83276f01d5f.mov